src/components/expanded-state/unique-token/UniqueTokenExpandedStateContent.js

```diff
@@ -115,7 +115,6 @@ const UniqueTokenExpandedStateContent = ({
             backgroundColor={asset.background}
```

Comment:
Didn't we want to have this small size to make this load faster when the big one is still fetching? If not, why did we have it?

Reply:
doing this in the underlying image component now